src/module/address/interface/address.interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface ICreateAddress {
+  country: string;
+  state: string;
+  city: string;
+  addresses: string[];
```

Comment:
one instance of Address represents 1 address, so in the future, we should have an array of addresses in the user. addresses .
If you want to create in a bulk check the documentation, the save method can receive an array of entities too.